src/test/java/io/meeds/qa/ui/steps/TasksSteps.java

```diff
@@ -716,6 +716,10 @@ public void setInSearchProjectField(String project) {
     tasksPage.setInSearchProjectField(project);
   }
 
+  public void setInSearchSpaceField(String space) {
+    tasksPage.setInSearchSpaceField(space);
+  }
+
```

Comment:
useless, try to reuse existing scenario definition :
https://github.com/Meeds-io/meeds-qa-ui/blob/c01e8c4626e2a571abc351cdedaf9d51f9000622/src/test/java/io/meeds/qa/ui/steps/definition/ManageSpaceStepDefinitions.java#L344-L347